packages/dapp/components/redeem/UcrNftRedeem.tsx

```diff
@@ -43,9 +42,9 @@ const UcrNftRedeem = () => {
   if (!deployedContracts || !managedContracts || !debtIds) return <span>· · ·</span>;
   if (debtIds.length === 0) return <span>No uCR-NFT coupons</span>;
 
-  async function fetchDebts(address: string, contract: DebtCoupon) {
```

Comment:
The custom typing `DebtCoupon` has been replaced with a generic `Contract` typing. @0xcodercrane and I had a discussion on this in Telegram and agreed that these should be fixed/switched back in the future for enhanced type safety. I understand that there were complications to get these custom typings stable with Foundry.

Includes all the other similar changes to this.

Reply:
created an issue for it.
https://github.com/ubiquity/ubiquity-dollar/issues/250